VideoEncoding/Encoding_PredefinedPreset/Program.cs

```diff
@@ -22,6 +21,12 @@ class Program
         private const string OutputFolder = @"Output";
         private const string TransformName = "AdaptiveBitrate";
         private const string DefaultStreamingEndpointName = "default";
+        private const string TokenType = "Bearer";
+        private const string BaseSourceUri = "https://nimbuscdn-nimbuspm.streaming.mediaservices.windows.net/2b533311-b215-4409-80af-529c3e853622/";
+        private const string FileSourceUri = "Ignite-short.mp4";
+
+        // Set this variable to true if you want to authenticate Interactively through the browser using your Azure user account
+        private const bool UseInteractiveAuth = false;
```

Comment:
Can you also note this in the root Readme.md file that this sample shows how to use Interactive login through the browser? I think that would be handy to call out in the root Readme table of samples and what they show.